modules/apps/adaptive-media/adaptive-media-image-impl-test/src/testIntegration/java/com/liferay/adaptive/media/image/internal/scaler/test/AMImageScalerRegistryImplTest.java

```diff
@@ -312,7 +312,7 @@ private ServiceRegistration<AMImageScaler> _registerAMImageScaler(
 		Dictionary<String, Object> properties = new Hashtable<>();
 
 		properties.put("service.ranking", serviceRanking);
-		properties.put("supportedMimeTypes", mimeType);
+		properties.put("mimeTypes", mimeType);
```

Comment:
You need to sort this alphabetically.

Reply:
Thanks @adolfopa, I've added a new commit fixing that.